aikau/src/main/resources/alfresco/services/SearchService.js

```diff
@@ -206,25 +209,26 @@ define(["dojo/_base/declare",
             }
             else
             {
-               sort = ((payload.sortField != null) ? payload.sortField : this.sort) + "|" + 
-                      ((payload.sortAscending != null) ? payload.sortAscending : this.sortAscending);
+               sort = ((payload.sortField) ? payload.sortField : this.sort) + "|" + 
+                      ((payload.sortAscending) ? payload.sortAscending : this.sortAscending);
             }
```

Comment:
This is a bit difficult to read with so many brackets and such repetition. Would you consider replacing with the below?

```
sort = (payload.sortField || this.sort) + "|" + (payload.sortAscending || this.sortAscending);
```